README.md

```diff
@@ -30,6 +30,7 @@ If you've already cloned without `--recursive`, run:
 ```
 git submodule init
 git submodule update
+cd coronavirus-data-sources && git submodule init && git submodule update
```

Comment:
Does this fix all the dang errors we've been having with submodules?!

Reply:
Hmm ok, but I wonder, can we pass `--recursive` to `git submodule init`? or maybe do `git submodule update --init --recursive`? Can you try fresh and see what works and doesn't require `cd`ing?

Reply:
I have removed the README patch from this branch for now.

>  Can you try fresh and see what works and doesn't require cding?

Will do and report back!

Reply:
In our CI scripts, we `git submodule update --init --recursive` which has been working well so far. Big issue is the submodule inside `coronavirus-data-sources` hence why we need to add `--recursive`